WalletWasabi.Tests/UnitTests/Wallet/SmartCoinSelectorTests.cs

```diff
@@ -100,38 +154,45 @@ public void PreferSameScript()
 	[Fact]
 	public void PreferMorePrivateClusterScript()
 	{
-		var keys1 = Enumerable.Range(0, 5)
-				.Select(_ => KeyManager.GenerateNewKey(new SmartLabel("Juan"), KeyState.Clean, false))
-				.ToList();
-
-		var juanCluster = new Cluster(keys1);
-
-		var coinsKnownByJuan = keys1
-			.ConvertAll(key =>
-			{
-				var coin = BitcoinFactory.CreateSmartCoin(key, 0.2m);
-				coin.HdPubKey.Cluster = juanCluster;
-				return coin;
-			});
-
-		var keys2 = Enumerable.Range(0, 2)
-				.Select(_ => KeyManager.GenerateNewKey(new SmartLabel("Juan"), KeyState.Clean, false))
-				.ToList();
-
-		var betoCluster = new Cluster(keys2);
+		var coinsKnownByJuan = GenerateSmartCoins(Enumerable.Repeat(("Juan", 0.2m), 5));
 
-		var coinsKnownByBeto = keys2
-			.ConvertAll(key =>
-			{
-				var coin = BitcoinFactory.CreateSmartCoin(key, 0.2m);
-				coin.HdPubKey.Cluster = betoCluster;
-				return coin;
-			});
+		var coinsKnownByBeto = GenerateSmartCoins(Enumerable.Repeat(("Juan", 0.2m), 2));
```

Comment:
This inherited a bug from the original code: 
```suggestion
		var coinsKnownByBeto = GenerateSmartCoins(Enumerable.Repeat(("Beto", 0.2m), 2));
```